app/scripts/migrations/078.test.js

```diff
@@ -0,0 +1,98 @@
+import migration78 from './078';
+
+describe('migration #78', () => {
+  it('should update the version metadata', async () => {
+    const oldStorage = {
+      meta: {
+        version: 77,
+      },
+    };
+
+    const newStorage = await migration78.migrate(oldStorage);
+    expect(newStorage.meta).toStrictEqual({
+      version: 78,
+    });
+  });
+
+  it('should remove the "showPortfolioToolip" property', async () => {
```

Comment:
Should probably add a test to ensure that in the absence of the showPortfolioTooltip key the state structure is the same.